### ISSUE TITLE
list index out of range when trying to predict

### ISSUE THREAD

**Issue by guilhermenoronha** [jidasheng/bi-lstm-crf#10]
Hello,

My code is getting the error of 'list index out of range' when I try to predict a sentence from the model. A analyzed your code but couldn't figure out what could it be.

Here is my code:

```
    model = WordsTagger(model_dir='name_model')
    tags, sequences = model([["meu", "amigo", "e", "senhor", "."]])
    print(tags)  
    print(sequences)
```

Here is the log:

```
`Traceback (most recent call last):

  File "C:\Users\guilh\OneDrive\Textos e Documentação\workspace\BI-LSTM-CRF Cartas\bilstmprocessor.py", line 30, in <module>
    predict_text()

  File "C:\Users\guilh\OneDrive\Textos e Documentação\workspace\BI-LSTM-CRF Cartas\bilstmprocessor.py", line 26, in predict_text
    tags, sequences = model([["meu", "amigo", "e", "senhor", "."]])

  File "C:\Users\guilh\anaconda3\envs\bilstm_cartas\lib\site-packages\bi_lstm_crf\app\predict.py", line 40, in __call__
    return tags, self.tokens_from_tags(sentences, tags, begin_tags=begin_tags)

  File "C:\Users\guilh\anaconda3\envs\bilstm_cartas\lib\site-packages\bi_lstm_crf\app\predict.py", line 64, in tokens_from_tags
    tokens_list = [_tokens(sentence, ts) for sentence, ts in zip(sentences, tags_list)]

  File "C:\Users\guilh\anaconda3\envs\bilstm_cartas\lib\site-packages\bi_lstm_crf\app\predict.py", line 64, in <listcomp>
    tokens_list = [_tokens(sentence, ts) for sentence, ts in zip(sentences, tags_list)]

  File "C:\Users\guilh\anaconda3\envs\bilstm_cartas\lib\site-packages\bi_lstm_crf\app\predict.py", line 56, in _tokens
    begins = [b for idx, b in enumerate(begins) if idx == 0 or ts[idx] != "O" or ts[idx - 1] != "O"]

  File "C:\Users\guilh\anaconda3\envs\bilstm_cartas\lib\site-packages\bi_lstm_crf\app\predict.py", line 56, in <listcomp>
    begins = [b for idx, b in enumerate(begins) if idx == 0 or ts[idx] != "O" or ts[idx - 1] != "O"]

IndexError: list index out of range`

```

I'd appreciate any help. Thanks in advance. Best regards.

**Comment by jidasheng** [comment]
Hello, I can't fix the problem with the information you provide. 
You can go inside the code in predict.py and print more info. 
More specifically, print all inputs of the function "tokens_from_tags", the value of "begins" before the crash.

**Comment by guilhermenoronha** [comment]
Please, check if the prints I've sent to you are enough. If not, let me know it. Best regards.

![print 1](https://user-images.githubusercontent.com/2208226/96996566-790fd580-1506-11eb-883c-31daa166b1ab.PNG)
![print 2](https://user-images.githubusercontent.com/2208226/96996745-ca1fc980-1506-11eb-952a-68382227f0db.PNG)
![print 3](https://user-images.githubusercontent.com/2208226/97000483-2c7bc880-150d-11eb-9487-adfd2bdb327d.PNG)

**Comment by jidasheng** [comment]
The bug is fixed in the new version 0.2.1. You can upgrade and try again.
Thanks for submitting the issue!

**Comment by guilhermenoronha** [comment]
Worked! Thx!
